patroni/api.py

```diff
@@ -673,6 +677,7 @@ def get_postgresql_status(self, retry=False):
                 'state': postgresql.state,
                 'postmaster_start_time': row[0],
                 'role': 'replica' if row[1] == 0 else 'master',
+                'sync_standby': 0,
```

Comment:
This one should probably be removed, in JSON it doesn't have so much value because absence of this field will automatically mean that the node is async.

```suggestion
```